### PR TITLE
【doc】Doc internationalization for plugin loadbalcner、flowcontrol、dynamic config、graceful and service registry.

### LINKED ISSUE #775: 【doc】doc internalization for loadbalcner、flowcontrol、dynamic config and service registry.
### What would you like to be added?

internalization doc for loadbalcner、flowcontrol、dynamic config and service registry.

### Why is this needed?

quick use

### PR DESCRIPTION
**Pull request by justforstudy-A**
What type of PR is this?

/doc

Which issue(s) this PR fixes:

Fixes #775 

Does this PR introduce a user-facing change?

No

Additional documentation e.g., usage docs, etc.:

loadbalcner、flowcontrol、dynamic config、graceful and service registry.

(1) loadbalcner
- [document.md](https://github.com/justforstudy-A/Sermant/blob/doc_en_zh/docs/user-guide/loadbalancer/document.md)
- [document-zh.md](https://github.com/justforstudy-A/Sermant/blob/doc_en_zh/docs/user-guide/loadbalancer/document-zh.md)
 
(2) flowcontrol
- [FAQ-zh.md](https://github.com/justforstudy-A/Sermant/blob/doc_en_zh/docs/user-guide/flowcontrol/FAQ-zh.md)
- [FAQ.md](https://github.com/justforstudy-A/Sermant/blob/doc_en_zh/docs/user-guide/flowcontrol/FAQ.md)
- [flowcontrol-zh.md](https://github.com/justforstudy-A/Sermant/blob/doc_en_zh/docs/user-guide/flowcontrol/flowcontrol-zh.md)
- [flowcontrol.md](https://github.com/justforstudy-A/Sermant/blob/doc_en_zh/docs/user-guide/flowcontrol/flowcontrol.md)

(3) dynamic config
- [document.md](https://github.com/justforstudy-A/Sermant/blob/doc_en_zh/docs/user-guide/dynamic-config/document.md)
- [document-zh.md](https://github.com/justforstudy-A/Sermant/blob/doc_en_zh/docs/user-guide/dynamic-config/document-zh.md)

(4) service registry
- [FAQ-zh.md](https://github.com/justforstudy-A/Sermant/blob/doc_en_zh/docs/user-guide/registry/FAQ-zh.md)
- [FAQ.md](https://github.com/justforstudy-A/Sermant/blob/doc_en_zh/docs/user-guide/registry/FAQ.md)
- [document.md](https://github.com/justforstudy-A/Sermant/blob/doc_en_zh/docs/user-guide/registry/document.md)
- [document-zh.md](https://github.com/justforstudy-A/Sermant/blob/doc_en_zh/docs/user-guide/registry/document-zh.md)
- [dubbo-registry-migiration.md](https://github.com/justforstudy-A/Sermant/blob/doc_en_zh/docs/user-guide/registry/dubbo-registry-migiration.md)
- [dubbo-registry-migiration-zh.md](https://github.com/justforstudy-A/Sermant/blob/doc_en_zh/docs/user-guide/registry/dubbo-registry-migiration-zh.md)
- [spring-cloud-registry-migiration.md](https://github.com/justforstudy-A/Sermant/blob/doc_en_zh/docs/user-guide/registry/spring-cloud-registry-migiration.md)
- [spring-cloud-registry-migiration-zh.md](https://github.com/justforstudy-A/Sermant/blob/doc_en_zh/docs/user-guide/registry/spring-cloud-registry-migiration-zh.md)

（5）graceful
- [document.md](https://github.com/justforstudy-A/Sermant/blob/doc_en_zh/docs/user-guide/graceful/document.md)
- [document-zh.md](https://github.com/justforstudy-A/Sermant/blob/doc_en_zh/docs/user-guide/graceful/document-zh.md)

### CODE REVIEW

File: docs/user-guide/registry/FAQ.md [hunk 1]
```diff
@@ -1,94 +1,102 @@
...
-|enableSpringRegister/enableDubboRegister|openMigration|作用|
+|enableSpringRegister/enableDubboRegister|openMigration|effect|
 |---|---|---|
-|true|true|开启spring cloud/dubbo迁移功能|
-|true|false|开启spring cloud/dubbo sc注册|
-|false|true|关闭注册插件|
-|false|false|关闭注册插件|
\ No newline at end of file
+|true|true|Enabling the Spring Cloud/Dubbo Migration Function|
+|true|false|Spring cloud/dubbo sc|
```
Comment (provenceee) [inline docs/user-guide/registry/FAQ.md]: maybe is "Enabling the Spring cloud/dubbo registration with the SC" ?

File: docs/user-guide/registry/FAQ.md [hunk 2]
```diff
@@ -1,94 +1,102 @@
...
+
+As shown in the following figure:
 
 ![No such extension org.apache.dubbo.registry.RegistryFactory by name sc](../../binary-docs/registry-faq-1.png)
 
-该错误原因有以下3种：
+The possible causes are as follows:
 
-1.1 宿主没有带agent启动。因为原生dubbo并不支持往sc注册，所以如果没带agent启动且配置的注册地址的协议为sc时，就会产生以上报错。
+1.1 The host is not started with an agent. The native dubbo does not support registration with the SC. Therefore, if the agent is not started and the protocol of the registration address is SC, the preceding error is reported.
```
Comment (provenceee) [inline docs/user-guide/registry/FAQ.md]: maybe is "The application is not started with an agent. The native dubbo does not support registration with the SC. Therefore, if the application is not started with an agent and the protocol of the registration address is SC, the preceding error is reported." ?

File: docs/user-guide/registry/FAQ.md (hunk 1 above)
Comment (justforstudy-A) [reply docs/user-guide/registry/FAQ.md]: Resolved

File: docs/user-guide/registry/FAQ.md (hunk 2 above)
Comment (justforstudy-A) [reply docs/user-guide/registry/FAQ.md]: Resolved

File: docs/user-guide/registry/document.md [hunk 3]
```diff
@@ -1,79 +1,77 @@
...
 
-需要强调的是，这个配置项的地址信息**不会使用**，只使用了协议名称sc（即ip地址不重要，只需要**sc://** 开头即可）。
-
-- **注意**：对于**存量**dubbo应用（即原本已经设置过dubbo本身注册中心地址的应用）**无需**进行这一步。
-
-## 结果验证
+Note that the address information of **this configuration item is not used**. Only the protocol name sc is used. (That is, the IP address is not important. **You only need to start with sc://**.)
 
-- 前提条件[正确打包Sermant](../../README.md)
+- **Note**: For **existing dubbo applications**, (i.e., applications where the address of the Dubbo registration center has been set) **This step is not required**.
```
Comment (provenceee) [inline docs/user-guide/registry/document.md]: maybe is "i.e., applications which"?

Comment (justforstudy-A) [reply docs/user-guide/registry/document.md]: Resolved

**Review by provenceee — DISMISSED** [review]

File: docs/user-guide/registry/dubbo-registry-migiration.md [hunk 4]
```diff
@@ -91,14 +89,16 @@ java -Dservicecomb.service.enableDubboRegister=true -Dserver.port=48021 -Ddubbo.
...
 ```
 
-注：为了便于测试，这里使用了-Dservicecomb.service.enableDubboRegister=true的方式打开了dubbo注册开关，如果使用了其它的方式打开了dubbo注册开关，则无需添加该参数；另外为了解决同一台服务器启动2个provider遇到的端口冲突问题，需要增加-Dserver.port=48021 -Ddubbo.protocol.port=48821参数，如果测试时2个provider在不同的服务器，则无需添加该参数。
+Notice: To facilitate the test, the Dubbo registration function is enabled in -Dservicecomb.service.enableDubboRegister=true mode. If the Dubbo registration function is enabled in other modes, you do not need to add this parameter. In addition, to solve the port conflict problem when two providers are started on the same server, you need to add the -Dserver.port=48021 -Ddubbo.protocol.port=48821 parameter. If the two providers are on different servers, you do not need to add this parameter.
+
+Please replace ${path} with the Sermant project path, replace x.x.x with the actual Sermant version number, and appName with the application name in the agent startup parameter, which is irrelevant to the registration parameter. The directory for running the command must be the directory where the JAR package of the demo application is located.
```
Comment (lilai23) [inline docs/user-guide/registry/dubbo-registry-migiration.md]: 改成`${path}`

File: docs/user-guide/loadbalancer/document.md [hunk 5]
```diff
@@ -1,130 +1,132 @@
...
+| spring-cloud-netflix-ribbon | availabilityFiltering                | availabilityFiltering / AVAILABILITY_FILTERING | spring cloud Edgware.x, spring cloud Finchley.x, spring cloud Greenwich.x, spring cloud Hoxton.x |
+| spring-cloud-netflix-ribbon | Response Time Weighting (Deprecated) | ResponseTimeWeighted / RESPONSE_TIME_WEIGHTED  | spring cloud Edgware.x, spring cloud Finchley.x, spring cloud Greenwich.x, spring cloud Hoxton.x |
+| spring-cloud-netflix-ribbon | Response time weighting              | weightedResponseTime / WEIGHTED_RESPONSE_TIME  | spring cloud Edgware.x, spring cloud Finchley.x, spring cloud Greenwich.x, spring cloud Hoxton.x |
+| spring-cloud-loadbalancer   | RoundRobin(default)                  | RoundRobin / ROUND_ROBIN                       | spring cloud Hoxton.SR10+, spring cloud 2020.0.x, spring cloud 2021.0.x |
+| spring-cloud-loadbalancer   | Random                               | Random / RANDOM                                | spring cloud Hoxton.SR10+, spring cloud 2020.0.x, spring cloud 2021.0.x |
 
-负载均衡基于配置中心进行动态配置，因此使用该能力需在配置中心配置对应负载均衡策略。负载均衡插件采用**流量标记+负载均衡规则**的方式规则，即配置一条规则需**同时配置两者**，下面分别介绍两项配置：
+## How to configure
```
Comment (lilai23) [inline docs/user-guide/loadbalancer/document.md]: 各级标题大写  How to Configure

File: docs/user-guide/loadbalancer/document.md [hunk 6]
```diff
@@ -1,130 +1,132 @@
...
 
-### 流量标记
+Load balancing is dynamically configured based on the configuration center. To use this capability, you need to configure the corresponding load balancing policy in the configuration center. The loadbalance plugin uses **traffic marking + loadbalance rules**. To configure a rule, you **need to configure both of them**. The following describes the two configurations:
 
-流量标记用于标记当前针对那些服务生效，若服务为空，则会应用与所有微服务，配置示例如下:
+### Traffic marking
```
Comment (lilai23) [inline docs/user-guide/loadbalancer/document.md]: 同上

File: docs/user-guide/flowcontrol/FAQ.md [hunk 7]
```diff
@@ -1,35 +1,41 @@
...
+## How Do I Determine Whether a Configuration Rule Takes Effect?
 
-## 熔断策略未生效的可能原因
+- Configure service scenarios and governance policies correctly in the configuration center. After the configuration, you can view the agent logs. Generally, in the logs folder in the startup path of the JAR package, view the sermant-x.log file and search for `has been` or the configured key name. If the found logs match the current time, the rule has taken effect.
 
-- 熔断生效有一定的前提，通常熔断从两个指标来判定：
-    - `异常比例`：即接口请求发生异常时所占比例，在规定时间内发生异常的比例大于配置的即会触发熔断；
-    - `慢调用比例`：即接口请求发生慢调用所占比例，设置熔断策略时需设置慢调用的阈值，例如100ms，则必须接口调用耗时超出100ms且超过配置的慢调用比例才可触发；
-- 因此针对以上两项指标，首先排查应用接口是否满足以上其中一个条件，且规则时间内调用超过最小调用数（minimumNumberOfCalls配置）才可触发。
+## Possible causes for the failure of the circuit breaker policy to take effect
```
Comment (lilai23) [inline docs/user-guide/flowcontrol/FAQ.md]: 大小写和前面一样保持一致

File: docs/user-guide/registry/document.md [hunk 8]
```diff
@@ -93,32 +91,32 @@ java -Dservicecomb.service.enableDubboRegister=true -javaagent:${path}\sermant-a
 java -Dservicecomb.service.enableDubboRegister=true -javaagent:${path}/sermant-agent-x.x.x/agent/sermant-agent.jar=appName=dubbo-provider -jar dubbo-provider.jar
 ```
 
-注：为了便于测试，这里使用了-Dservicecomb.service.enableDubboRegister=true的方式打开了dubbo注册开关，如果使用了其它的方式打开了dubbo注册开关，则无需添加该参数。
+Note: To facilitate the test, the DUBBO registration function is enabled in -Dservicecomb.service.enableDubboRegister=true mode. If the DUBBO registration function is enabled in other modes, you do not need to add this parameter.
 
-其中${path}需要替换为Sermant工程路径，x.x.x需要替换为Sermant实际版本号，appName为agent启动参数中的应用名，与注册参数无关，执行命令的目录需要为demo应用的jar包目录。
+Replace ${path} with the Sermant project path, replace x.x.x with the actual Sermant version number, and appName with the application name in the agent startup parameter, which is irrelevant to registration parameters. The directory for running commands must be the JAR package directory of the demo application.
```
Comment (lilai23) [inline docs/user-guide/registry/document.md]: 同上

File: docs/user-guide/registry/dubbo-registry-migiration.md (hunk 4 above)
Comment (justforstudy-A) [reply docs/user-guide/registry/dubbo-registry-migiration.md]: Resolved

File: docs/user-guide/loadbalancer/document.md (hunk 5 above)
Comment (justforstudy-A) [reply docs/user-guide/loadbalancer/document.md]: Resolved

File: docs/user-guide/flowcontrol/FAQ.md (hunk 7 above)
Comment (justforstudy-A) [reply docs/user-guide/flowcontrol/FAQ.md]: Resolved

File: docs/user-guide/registry/document.md (hunk 8 above)
Comment (justforstudy-A) [reply docs/user-guide/registry/document.md]: Resolved

**Review by provenceee — DISMISSED** [review]

File: docs/user-guide/loadbalancer/document.md (hunk 6 above)
Comment (justforstudy-A) [reply docs/user-guide/loadbalancer/document.md]: Resolved

**Review by lilai23 — DISMISSED** [review]

**Review by xuezechao1 — DISMISSED** [review]

File: docs/user-guide/dynamic-config/document-zh.md [hunk 9]
```diff
@@ -0,0 +1,222 @@
...
+service.meta.version=1.0.0
+# serviceComb 命名空间
+service.meta.project=default
+# 环境
+service.meta.environment=development
+# 自定义标签，按需配置，用于后续的配置订阅
+service.meta.customLabel=public
+service.meta.customLabelValue=default
+```
+-  应用配置：由`service.meta.application`与`service.meta.environment`组成， 对应的`group`为`app=default&environment=development`
```
Comment (luanwenfei-venus) [inline docs/user-guide/dynamic-config/document-zh.md]: 前面多了一个空格

**Review by luanwenfei-venus — DISMISSED** [review]

File: docs/user-guide/dynamic-config/document-zh.md (hunk 9 above)
Comment (justforstudy-A) [reply docs/user-guide/dynamic-config/document-zh.md]: Resolved

**Review by provenceee — DISMISSED** [review]

File: docs/README-zh.md [hunk 10]
```diff
@@ -85,11 +85,13 @@ k8s环境下，Sermant支持通过sermant-injector组件实现宿主应用自动
...
 |[后端模块介绍](user-guide/backend.md)|使用手册|
 |[容器化部署指导手册](user-guide/injector.md) |使用手册|
-|[限流降级功能介绍](user-guide/flowcontrol/flowcontrol.md)|使用手册|
-|[动态配置功能介绍](user-guide/dynamic-config/document.md)|使用手册|
-|[服务注册功能介绍](user-guide/registry/document.md)|使用手册|
+|[限流降级功能介绍](user-guide/flowcontrol/flowcontrol-zh.md)|使用手册|
+|[负载均衡功能介绍](user-guide/loadbalancer/document-zh.md)|使用手册|
+|[动态配置功能介绍](user-guide/dynamic-config/document-zh.md)|使用手册|
+|[服务注册功能介绍](user-guide/registry/document-zh.md)|使用手册|
+|[优雅上下线功能介绍](user-guide/graceful/document-zh.md)||
```
Comment (robotLJW) [inline docs/README-zh.md]: 缺少使用手册

Comment (justforstudy-A) [reply docs/README-zh.md]: Resolved

**Review by lilai23 — APPROVED** [review]

**Review by luanwenfei-venus — APPROVED** [review]

**Review by provenceee — APPROVED** [review]

**Pull request merged**